src/basilisp/contrib/sphinx/domain.py

```diff
@@ -135,6 +135,9 @@ def handle_signature(self, sig: str, signode: desc_signature) -> Tuple[str, str]
         """Subclasses should implement this themselves."""
         return NotImplemented
 
+    def get_index_text(self, modname: str, name: Tuple[str, str]) -> str:
+        return "TODO"
```

Comment:
Given that this alert appeared on you due to no changes of your own, I would advise suppressing the PyLint errors for now. They seem challenging to handle quickly without vastly increasing the scope of this PR and they don't seem to actually be breaking the ReadTheDocs build.

Reply:
Done, I had to use a double suppression because assignment-from-no-return does not trigger for pylint in python 3.8 ...